app/templates/font-chooser-item.hbs

```diff
@@ -1,3 +0,0 @@
-<div style={{view.style}}>
```

Comment:
This just seems unused? I looked for `font-chooser-item` in the codebase and in Iverson and I agree it isn't used at a glance.